### ISSUE TITLE
Multiple doc pages

### ISSUE THREAD

**Issue by vitodsk** [tiangolo/fastapi#3220]
If I have a bigger API which consists of different "sub-api", can I have multiple doc pages (one for each "sub-api") ?

Thanks

**Comment by Hultner** [comment]
Yes this is possible, I currently do this in a couple of applications. 

I achieve this by creating multiple `FastAPI(...)` instances and mounting them in the primary app.

Something like this
```python
app = FastAPI(title="Primary app", ...)

# Set up your routers, middleware, etc

sub_api = FastAPI(
    title="Sub API",
    openapi_url="/sub-api/openapi.json"   # The sub-route you use
    ...
)
# Set up your sub api
sub_api.include_router(...)

# You can now mount your sub api apps in your primary app.
app.mount(path="/sub-api", app=sub_api)
```

This way you can find the docs for the sub-api at http://localhost:8000/sub-api/docs and the primary docs at http://localhost:8000/docs, assuming you use localhost:8000 for development.

**Comment by tiangolo** [comment]
Thanks for the help here @Hultner ! 👏  🙇 

If that solves the original problem, then you can close this issue @vitodsk  ✔️

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
